documentation/src/main/asciidoc/topics/ref_hotrod_listeners.adoc

```diff
@@ -511,10 +513,6 @@ content has been iterated over, events are received as normal, as cache
 updates are received.  If the cache is clustered, the entire cluster wide
 contents are iterated over.
 
-`includeCurrentState` also controls whether state is received when the node
```

Comment:
why was this sentence removed? 
AFAIK, it is true: https://github.com/infinispan/infinispan/blob/master/client/hotrod-client/src/main/java/org/infinispan/client/hotrod/annotation/ClientListener.java#L64

Reply:
it seems like the following section "Listener Failure Handling" covers it in more detail so the sentence was redundant/repetitive.